Tests/SwiftSyntaxBuilderTest/ExpressibleBuildablesTests.swift

```diff
@@ -0,0 +1,95 @@
+import XCTest
+import SwiftSyntax
+
+import SwiftSyntaxBuilder
+
+final class ExpressibleBuildablesTests: XCTestCase {
+    func testExpressibleAsMemberDeclListItem() {
+        let myStruct = StructDecl(identifier: "MyStruct", members: MemberDeclBlock(membersBuilder: {
+            VariableDecl(.var, name: "myFirstVar", type: "Int")
+
+            VariableDecl(letOrVarKeyword: .let, bindingsBuilder: {
+                PatternBinding(pattern: IdentifierPattern(identifier: .identifier("mySecondLet")),
+                               typeAnnotation: TypeAnnotation(type: SimpleTypeIdentifier("CGFloat")))
+            })
+
+            MemberDeclListItem(decl: VariableDecl(letOrVarKeyword: .let, bindingsBuilder: {
```

Comment:
Do you want to explicitly test we can put a `MemberDeclListItem` here? If so, I think it would be good to add a comment explaining your intention.

Reply:
My intention is to make sure they can be combined and showcasing it.
I will add a comment

Reply:
Thats’s good! Just wanted to make sure we’re on the same page what should be tested here.